Set Up Verbose Logging Helper Libraries

Set up a helper library that has functions to print different types of timestamped logs. 


## Expected Code Usage
```py
from logging import error, info, debug 

error("Something very bad happened")
info("Just to let you know btw.")
debug("Do you really wanna about all this?")
```

## Expected Output
```
[ERROR] [02/03/2025 15:36] Something very bad happened
[INFO] [02/03/2025 15:39] Just to let you know btw.
[DEBUG] [02/03/2025 15:45] Do you really wanna about all this?
```

## Additional Information
**Timestamp format:** DD/MM/YYYY HH:MM

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,5 @@
+log.txt
+
 # Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]

--- a/bot.py
+++ b/bot.py
@@ -1,0 +1,18 @@
+import discord
+import os 
+from dotenv import load_dotenv
+from logger import error, info, debug 
+load_dotenv() 
+bot = discord.Bot()
+
+@bot.event
+async def on_ready():
+    info(f"{bot.user} is ready and online!")
+
+@bot.slash_command(name="hello", description="Say hello to the bot")
+async def hello(ctx: discord.ApplicationContext):
+    await ctx.respond("Hey!")
+
+bot.run(os.getenv('TOKEN'))
+
+

--- a/logger.py
+++ b/logger.py
@@ -1,0 +1,29 @@
+# Module to enable timestamped logs
+from datetime import datetime
+from pytz import utc, timezone
+
+class LOG_LEVEL:
+    ERROR = 'ERR'
+    INFO  = 'INF'
+    DEBUG = 'DBG'
+
+def _generate_timestamp() -> str:
+    date_format = "%d-%b-%Y %H:%M:%S"
+    date = datetime.now(tz=utc)
+    date = date.astimezone(timezone('US/Pacific'))
+    return date.strftime(date_format)
+
+def _log(log_level: str, *values: object, sep: str | None = " ", end: str | None = "\n") -> None:
+    print(f"[{log_level}][{_generate_timestamp()}]", *values, sep=sep, end=end)
+    with open('log.txt', 'a') as log_file:
+        print(f"[{log_level}] [{_generate_timestamp()}]", *values, sep=sep, end=end, file=log_file)
+
+def error(*values: object, sep: str | None = " ", end: str | None = "\n") -> None:
+    _log(LOG_LEVEL.ERROR, *values, sep=sep, end=end)
+
+def info(*values: object, sep: str | None = " ", end: str | None = "\n") -> None:
+    _log(LOG_LEVEL.INFO, *values, sep=sep, end=end)
+
+def debug(*values: object, sep: str | None = " ", end: str | None = "\n") -> None:
+    _log(LOG_LEVEL.DEBUG, *values, sep=sep, end=end)
+

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,0 +1,3 @@
+py-cord
+dotenv
+pytz

